en/tutorials/linux_sbus.md

```diff
@@ -0,0 +1,35 @@
+# Sbus Driver for linux（drivers/linux_sbus）
+This driver allows the autopilot to fetch data from the futaba sbus receiver via serial port, up to 16 channels.  
+Onboard serial, usbtty, and other types of serial ports are supported.  
+All platforms running linux system are supported.  
+
+## Required components
+
+NPN transistor * 1  
+10K resistance * 1  
+1K resistor * 1  
+
+## Optional components
+Usbtty *1  
```

Comment:
I suppose you are making reference to a "USB to TTL Serial Cable/Converter". Needs to be more clear here.